src/identity/checkAddress.ts

```diff
@@ -0,0 +1,3 @@
+export function checkAddress(hash: string, length: number): hash is `0x${string}` {
```

Comment:
`checkAddress` as name feels very vague, what does this exactly?

Reply:
Maybe `checkHashLength` is more appropriate. Giving early errors to the devs if they used an incorrect schemaId